devicemodel/hw/platform/acpi/acpi_parser.c

```diff
@@ -0,0 +1,131 @@
+/*-
+ * Copyright (c) 2011 NetApp, Inc.
```

Comment:
We seem to have a number of ACRN-specific modifications that were made to this file. Shouldn't the Intel Copyright notice have been added?